docs/monitoring/quality.rst

```diff
@@ -0,0 +1,46 @@
+.. |MonitorWatchDog| image:: ../images/MonitorWatchDog.jpg
+
+Data Quality
+============
+
+There are many different definitions of data quality and many different mechanisms that can be used within Fledge to test the quality of the data ingested into Fledge.
+
+There are several criteria that can be used to determine poor quality data, the very simplest mechanisms are things such as data falling outside of expected limits, this can be very easily detected by using the threshold notification filter to detect over or under limit values. The next level of data quality monitoring it to look for outliers by observing patterns in the data or by viewing the statistical variations within the data.
```

Comment:
- Start new sentence at "The very simplest mechanisms are things..."
- Start new sentence at "This can be very easily detected by using..."
- "...data quality monitoring it to look..." -> "...data quality monitoring is to look..."